.github/actions/simplecov-report/action.yml

```diff
@@ -18,7 +18,7 @@ inputs:
     default: "coverage/.last_run.json"
     required: false
   token:
-    description: "Github token"
+    description: "GitHub token"
```

Comment:
Brand.

I'm not sure where this thing comes from, but the project has a number of typos, and assuming it isn't original to this repo, I'd like to send a PR for it.

I've actively dropped a couple of changes to associated files.